front/src/containers/Authentication/register.js

```diff
@@ -18,42 +18,83 @@ import {
   Container,
   Logo,
   InputField,
-  HyperLink
-} from './style'
+  HyperLink,
+  ContainerLogo,
+  ContainerView,
+  ContainerItem,
+  Icon,
+  Button,
+  Text
+} from './styleRegister'
+import logo from '../../assets/logo.png'
 
 const RegisterScreen = (props) => {
 
   return (
     <Container>
-      <Logo>Globo Plus</Logo>
 
-      <View>
-        <Text>Nome de usuário</Text>
-        <InputField
-          value={props.values.username}
-          onChangeText={text => props.setFieldValue('username', text)} />
+      <ContainerLogo>
+        <Logo source={logo} />
+      </ContainerLogo>
 
-        <Text>Email</Text>
-        <InputField
-          value={props.values.email}
-          onChangeText={text => props.setFieldValue('email', text)} />
+      <ContainerView>
+        <ContainerItem>
+          <InputField
+            placeholder='Digite seu email'
+            value={props.values.username}
+            onChangeText={text => props.setFieldValue('username', text)} />
 
-        <Text>Senha</Text>
-        <InputField
-          value={props.values.password}
-          onChangeText={text => props.setFieldValue('password', text)} />
+          <Icon>
+            <IconEf
+              name='email'
+              color='#fff'
+              size={35}
+            />
+          </Icon>
+        </ContainerItem>
 
-        <Button
-          onPress={props.handleSubmit}
-          title='Cadastrar' />
+        <ContainerItem>
+          <InputField
+            placeholder='Digite seu login'
+            value={props.values.email}
+            onChangeText={text => props.setFieldValue('email', text)} />
+
+          <Icon>
+            <IconUser
+              name='user'
+              color='#fff'
+              size={35}
+            />
+          </Icon>
+        </ContainerItem>
+
+        <ContainerItem>
+          <InputField
+            placeholder='Digite sua senha'
+            value={props.values.password}
+            secureTextEntry
+            onChangeText={text => props.setFieldValue('password', text)} />
+
+          <Icon>
```

Comment:
You can apply the style of it in own "IconKey", no? You can test it.
I see that was added only a position: absolute; and a right property